charts/polkadot-registrar-challenger/templates/statefulset.yaml

```diff
@@ -36,8 +36,10 @@ spec:
         #   initialDelaySeconds: 30
         #   timeoutSeconds: 3
         volumeMounts:
-        - name: data
-          mountPath: /data
+        - name: matrix-store
+          mountPath: /tmp/matrix_db
+        - name: registrar-db
+          mountPath: /tmp/registrar_db  
```

Comment:
The application is not yet configurable. Then, these two mountPaths will become dynamic